apps/sms/services/js/message_manager.js

```diff
@@ -510,9 +510,6 @@ var MessageManager = {
         _isMessageBelongTo1to1Conversation(number, message);
       if (isMessageInThread) {
         threadId = message.threadId;
-        // we need to set the current threadId,
-        // because we start sms app in a new window
-        Threads.registerMessage(message);
```

Comment:
But the behavior in here seems a little bit different then `ensureThreadRegistered` because here we register the message that is not registered yet. But in `ensureThreadRegistered` we simply  register the first returned message without any checking. Shouldn't we do the same check in the `ensureThreadRegistered`?